pkg/composer/down.go

```diff
@@ -31,8 +34,17 @@ type DownOptions struct {
 }
 
 func (c *Composer) Down(ctx context.Context, downOptions DownOptions) error {
-	for _, svc := range c.project.Services {
-		if err := c.downService(ctx, svc, downOptions.RemoveVolumes); err != nil {
```

Comment:
`downService()` is no longer used? Can we remove `downService()`?

Reply:
Sure.